File: PV239/seminar5/src/main/java/cz/pv239/seminar5/fragment/SettingsFragment.java
package cz.pv239.seminar5.fragment;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.preference.PreferenceFragmentCompat;
import cz.pv239.seminar5.R;

public class SettingsFragment
        extends PreferenceFragmentCompat {

    @NonNull
    public static SettingsFragment newInstance() {
        return new SettingsFragment();
    }

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        addPreferencesFromResource(R.xml.settings);
    }
}

File: PV239/seminar5/src/main/java/cz/pv239/seminar5/fragment/SettingsFragment.kt
package cz.pv239.seminar5.fragment

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import cz.pv239.seminar5.R

class SettingsFragment : PreferenceFragmentCompat() {

    companion object {

        fun newInstance(): SettingsFragment {
            return SettingsFragment()
        }
    }

    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        addPreferencesFromResource(R.xml.settings)
    }
}
